src/lib/layout/layout.ts

```diff
@@ -41,7 +41,7 @@ export type ScanResult = {
   }
   sourceRoot: string
   projectRoot: string
-  schemaModules: string[]
+  nexusModules: string[]
```

Comment:
Because Nexus will now imports **any** modules that contains a `nexus` import, I decided to rename it to `nexusModules`. We could as well just call it `modules`